src/LoginPage/LoginForm.jsx

```diff
@@ -1,46 +1,55 @@
 import { memo } from 'react';
 
-export default memo(({ onClick, isLogin }) => ((
-  <form>
-    {isLogin ? null : (
-      <>
-        <input
-          name="first"
-          type="text"
-          placeholder="First Name"
-        />
-        <input
-          name="last"
-          type="text"
-          placeholder="Last Name"
-        />
-      </>
-    )}
-    <input
-      name="email"
-      type="email"
-      placeholder="Email"
-    />
-    <input
-      name="password"
-      type="password"
-      placeholder="Password"
-      autoComplete="off"
-    />
-    {isLogin ? (
-      <button type="submit">Log In</button>)
-      : (<button type="submit">Sign Up</button>)}
+export default memo(({ isLogin, onClick, onChange }) => {
+  function handleChange(event) {
+    const { target: { name, value } } = event;
+    onChange({ name, value });
+  }
 
-    {isLogin ? (<p>Don&apos;t have an account?</p>) : (
-      <p>
-        Passwords must be longer than 7 chars and include numbers
-      </p>
-    )}
-    <button
-      type="button"
-      onClick={onClick}
-    >
-      {isLogin ? 'Sign Up' : 'Log In' }
-    </button>
-  </form>
-)));
+  return (
+    <form>
+      {isLogin ? null : (
+        <>
+          <input
+            name="first"
+            type="text"
+            placeholder="First Name"
+          />
+          <input
+            name="last"
+            type="text"
+            placeholder="Last Name"
+          />
+        </>
+      )}
+      <input
+        name="email"
+        type="email"
+        placeholder="Email"
+        onChange={handleChange}
+      />
+      <input
+        name="password"
+        type="password"
+        placeholder="Password"
+        autoComplete="off"
+        onChange={handleChange}
+      />
+      {isLogin ? (
+        <button type="submit">Log In</button>)
+        : (<button type="submit">Sign Up</button>)}
```

Comment:
```suggestion
<button type="submit">{isLogin ? 'Log In' : 'Sign Up'}</button>
```

아래쪽 버튼처럼 작성해주시는게 렌더링, 가독성 측면에서 더 좋을 거 같아요!
아래 p태그도 한번 고쳐주세용!ㅎㅎ

Reply:
이 부분들을 저렇게 고쳤어야 했는데 몰랐네요
덕분에 코드가 훨씬 가독성이 좋아졌습니다!
🙏🏼